jmclient/jmclient/configure.py

```diff
@@ -241,8 +241,8 @@ def jm_single():
 # spends from unconfirmed inputs, which may then get malleated or double-spent!
 # other counterparties are likely to reject unconfirmed inputs... don't do it.
 
-#options: self, random-peer, not-self (note: random-maker is not currently supported).
-# self = broadcast transaction with your own ip
+# options: self, random-peer, not-self (note: currently, ONLY 'self' works).
+# self = broadcast transaction with your bitcoin core's ip
```

Comment:
Maybe change "bitcoin core" to "bitcoin node"? AFAIK bcoin (JavaScript full node implementation by Purse.IO) is RPC-compatible with bitcoind, and also could be used, for example. Also, one could use Bitcoin Knots instead of Bitcoin Core.

Reply:
Good point. Changed!